jenkins-build-driver/src/main/java/org/jboss/pnc/jenkinsbuilddriver/BuildJob.java

```diff
@@ -35,19 +44,19 @@ public boolean configure(RunningEnvironment runningEnvironment, boolean override
         try {
             job = jenkinsServer.getJob(jobName);
         } catch (IOException e) {
-            throw new BuildDriverException("Cannot check for existing job.", e);
+            log.info("No job exists, so there will be created new one.");
```

Comment:
But if you rethrow it you never create a new job, the method jenkinsServer.getJob() can have two exceptions one as HttpResponseException which is still in IOException inheritance but there is other which are not caught in  jenkinsServer.getJob() method and that could be blocker for the case there is not job or there is something with Jenkins connection.